configure.ac

```diff
@@ -158,7 +165,9 @@ dnl Now make the Makefiles
 dnl ######################################################################
 
 AC_CONFIG_FILES([Makefile
-                tests/acceptance/Makefile])
+                package_modules/Makefile
+                tests/acceptance/Makefile
+                tests/unit/Makefile])
```

Comment:
Can you use make variable expansion ? $(SUBDIRS:%=%/Makefile) would save the need for changes if you add more SUBDIRS.